tests/phpunit/tests/ajax/CropImage.php

```diff
@@ -0,0 +1,150 @@
+<?php
+
+/**
+ * Admin Ajax functions to be tested.
+ */
+require_once ABSPATH . 'wp-admin/includes/ajax-actions.php';
+
+require_once ABSPATH . 'wp-admin/includes/class-wp-filesystem-base.php';
+require_once ABSPATH . 'wp-admin/includes/class-wp-filesystem-direct.php';
+
+/**
+ * Class for testing ajax crop image functionality.
+ *
+ * @group ajax
+ */
+class Tests_Ajax_CropImage extends WP_Ajax_UnitTestCase {
+
+	/**
+	 * @covers wp_ajax_crop_image
+	 */
+	public function test_it_copies_metadata_from_original_image() {
+
+		// Become an administrator.
+		$this->_setRole( 'administrator' );
+
+		$attachment = $this->create_attachment( true );
+		$this->prepare_post( $attachment );
+
+		// Make the request.
+		try {
+			$this->_handleAjax( 'crop-image' );
+		} catch ( WPAjaxDieContinueException $e ) {
+		}
+
+		$response = json_decode( $this->_last_response, true );
+		$this->validate_response( $response );
+
+		$cropped_attachment = get_post( $response['data']['id'] );
+		$this->assertInstanceOf( WP_Post::class, $cropped_attachment );
+		$this->assertNotEmpty( $attachment->post_title );
+		$this->assertNotEmpty( $cropped_attachment->post_title );
+		$this->assertSame( $attachment->post_title, $cropped_attachment->post_title );
```

Comment:
There's a few assertions here in the same test.  In this instance, it's best to include the `$message` parameter when you have multiple assertions in a test.  See https://make.wordpress.org/core/handbook/testing/automated-testing/writing-phpunit-tests/#using-assertions

Reply:
I agree. Thank you for the feedback.
Fixed in f96f9d31420ce77768c94bddfeb9babedafaa716